plugins/community/pseudomuto-doc/v1.5.1/buf.plugin.yaml

```diff
@@ -0,0 +1,6 @@
+version: v1
+name: buf.build/community/pseudomuto-doc
+plugin_version: v1.5.1
+source_url: https://github.com/pseudomuto/protoc-gen-doc
+description: Documentation generator plugin for Google Protocol Buffers.
+spdx_license_id: MIT
```

Comment:
We've recently added support for license URLs to these files. Can you add:

```
license_url: https://github.com/pseudomuto/protoc-gen-doc/blob/v1.5.1/LICENSE.md
```

here? This will allow the build to pass once rebased/merged to pull in latest changes.